### PR TITLE
feature: research page card redesign

### LINKED ISSUE #2292: [refactor] Remove direct usage of Theme from "ResearchListItem.tsx"
Remove the following

```ts
// TODO: Remove direct usage of Theme
import { preciousPlasticTheme } from 'oa-themes'
const theme = preciousPlasticTheme.styles
```

At the moment we are importing the theme object directly and accessing its style properties. We should avoid doing this and instead make use of `sx` property available on all ThemeUI components to load these values out of the theme definition.

Read more about it here: 
> The `sx` prop lets you add any valid CSS to an element, while using values from your theme to keep styles consistent. You can think of the style object that the `sx` prop accepts as a _superset_ of CSS.
https://theme-ui.com/sx-prop

The aim should be for there to be no visual differences. However there may be circumstances where this would introduce better visual consistency across the UI. Feel free to reach out on [Discord ](https://discord.com/channels/586676777334865928/938781727017558018) if you think you have found one of those scenarios.

Here's an example where the same change has been made, although I would call out that this specific file required more work than most 🤗  https://github.com/ONEARMY/community-platform/pull/2282

### LINKED ISSUE #2450: [feature request] Research page card new design
**Is your feature request related to a problem? Please describe.**

You may find it difficult to understand the cards on the Research page since there are many elements as you scroll, namely the right portion with the three numbers (Useful, comments, and updates).

**Describe the solution you'd like**

In order to check whether there was something similar or how the issue was solved, I also looked at the other pages.
One solution would be to relocate the top icons from the "table header" to each card and add a tooltip that would explain each icon's meaning when the mouse is hovered over it. This is how we address the issue of visibility.

**Describe alternatives you've considered**

Why pagination isn't utilized for "tables" so we may restrict the number of entries per page is one thing I don't understand.

Therefore, pagination is used to reduce the amount of entries

**Additional context**
I quickly mocked up the version I was envisioning.
Group 98

Additionally, I'm not sure why the user needs to know how often it was updated when he only needs to know when the most recent update occurred.

### PR DESCRIPTION
**Pull request by BaltacMihai**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Developer experience (improves developer workflows for contributing to the project)

## Description

Place the top icons on each individual card instead of the "table header". I used a placeholder until we get an icon for the "Updates" sign.
Removes direct usage of the theme object, instead accessing it within the sx property.

--
Additionally, I changed the empty state symbol from "-" to "0" because the user will now understand it better. In the old form, the user might have wondered if the data had loaded or not.

## Git Issues

Closes #2450 
Closes #2292

## Screenshots/Videos

image

### CODE REVIEW

File: src/pages/Research/Content/ResearchListItem.tsx [hunk 1]
```diff
@@ -1,46 +1,27 @@
...
   const collaborators = item['collaborators'] || []
   const usefulDisplayCount =
-    item.votedUsefulCount > 0 ? item.votedUsefulCount : '-'
+    item.votedUsefulCount > 0 ? item.votedUsefulCount : '0'
+
+  const _commonStatisticStyle = {
+    display: 'flex',
+    alignItems: 'center',
+    fontSize: [1, '16px', '16px'],
+  }
```
Comment (AlfonsoGhislieri) [inline src/pages/Research/Content/ResearchListItem.tsx:24]: **Thought** interesting to see that we dont have 16 in the fontsize array, I'd suggest adding it, but I'm not sure it gets used elsewhere in the codebase and that would require updating the other fontSizes..

Comment (BaltacMihai) [reply src/pages/Research/Content/ResearchListItem.tsx:24]: Only three files, including this one, use it, as far as I can tell. I suggest using a pre-existing value, such as 14 or 18 pixels, for linearity.

Comment (AlfonsoGhislieri) [reply src/pages/Research/Content/ResearchListItem.tsx:24]: That sounds good to me, much better for future maintenance as well if we can standardise fontsizes across the platform!

Comment (BaltacMihai) [reply src/pages/Research/Content/ResearchListItem.tsx:24]: I designed 2 types of icons that I think maybe fit.

Group 104

A @davehakkens I posted some comments on Figma about the overall icons

**Comment by davehakkens** [comment]
Looks good!
Here you can find the icon @BaltacMihai 
research-icon

**Comment by davehakkens** [comment]
Just noticed the font-size of the numbers is quite large, 18px.
Could this be reduced to 14px @BaltacMihai  (like on the how-to overview page)

**Pull request merged**